### ISSUE TITLE
Glass Mode border

### ISSUE THREAD

**Issue by swessel87** [gileli121/WindowTop#355]
I love this app and the glass mode that is available however if I lower the blur setting all the way down the is a border that I cannot seem to get rid of using any options. Is this intended or am I missing something, or possibly missing something?

**Comment by gileli121** [comment]
@swessel87 
Indeed, it was a known issue that I couldn't resolve in the past, but it's now fixable. The Windows API function used by the feature had a limitation that resulted in an additional border. However, they've introduced an option to this API function that enables the removal of the border, allowing me to address the problem.

I'll include this bug in the to-do list and start working on a fix.
Thank you for the report.
